src/components/molecules/VenueRoomItem/VenueRoomItem.tsx

```diff
@@ -0,0 +1,179 @@
+          bannerImageFile: fileList,
+          bannerImageUrl: "",
+          logoImageUrl: "",
+          mapBackgroundImageUrl: "",
+          logoImageFile: fileList,
+          rooms: [],
+        };
+
+        await createVenue_v2(venueInput, user);
+      }
+
+      await createRoom(roomData, venueId, user);
+      hideModal();
+    } catch (err) {
+      console.error(err);
+    }
+  }, [getValues, hideModal, template, user, venueId]);
+
+  return (
+    <>
+      <Modal show={isModalVisible} onHide={hideModal}>
+        <Modal.Body>
+          <Form onSubmit={handleSubmit(addRoom)}>
+            <Form.Row>
+              <div className="room-edit-modal__input">
+                <Form.Label>Room title</Form.Label>
+                <Form.Control
+                  disabled={loading}
+                  type="text"
+                  ref={register}
+                  name="roomTitle"
+                  placeholder="Room title"
+                  custom
+                />
+                {errors.roomTitle && (
+                  <span className="input-error">
+                    {errors.roomTitle.message}
+                  </span>
+                )}
+              </div>
+            </Form.Row>
+
+            {template && (
+              <Form.Row>
+                <div className="room-edit-modal__input">
+                  <Form.Label>Venue name</Form.Label>
+                  <Form.Control
+                    disabled={loading}
+                    type="text"
+                    ref={register}
+                    name="venueName"
+                    placeholder="Venue name"
+                    custom
+                  />
+                  {errors.venueName && (
+                    <span className="input-error">
+                      {errors.venueName.message}
+                    </span>
+                  )}
+                </div>
+              </Form.Row>
+            )}
+
+            {!template && (
+              <Form.Row>
+                <div className="room-edit-modal__input">
+                  <Form.Label>Room url</Form.Label>
+                  <Form.Control
+                    disabled={loading}
+                    type="text"
+                    ref={register}
+                    name="roomUrl"
+                    placeholder="Room url"
+                    custom
+                  />
+                  {errors.roomUrl && (
+                    <span className="input-error">
+                      {errors.roomUrl.message}
+                    </span>
+                  )}
+                </div>
+              </Form.Row>
+            )}
+
+            <Button disabled={loading} title="Add room" type="submit">
+              Add room
+            </Button>
+          </Form>
+        </Modal.Body>
+      </Modal>
+      <div className="Spaces__venue-room" onClick={showModal}>
+        <div
+          className="Spaces__room-external-link"
+          style={{ backgroundImage: `url(${icon})` }}
```

Comment:
`useMemo`? `useStyle`?

Reply:
Fixed in: https://github.com/sparkletown/sparkle/commit/d2bc2049fe72c651d807803b6b568d74a74764f6